src/Network/Network/ApplicationGateway/Probe/AzureApplicationGatewayProbeConfigBase.cs

```diff
@@ -74,6 +74,12 @@ public class AzureApplicationGatewayProbeConfigBase : NetworkBaseCmdlet
         [ValidateRange(0, int.MaxValue)]
         public int MinServers { get; set; }
 
+        [Parameter(
+           Mandatory = false,
+           HelpMessage = "Port that is used for probing the backend server")]
+        [ValidateRange(1, 65535)]
+        public int? Port { get; set; }
```

Comment:
This parameter should be non-nullable. We only allow nullable for Update cmdlets (which this one doesn't have).

Reply:
@abjai @number213  can you please fix it quick?

Reply:
@MiYanni  The valid range of Port is from 1 to 65535. In case anyone not provide this value in the new powershell command, it will deserialize to 0 which will be wrong and validation error will be thrown. Hence, it need to be nullable.

I tried making it non nullable at the starting of PR but it lead to failure of other tests with validation exception of Port

Reply:
@MiYanni I've added validation in code to make sure model's `Port` would be null, if cmdlets `Port` wasn't set